backend/packages/Upgrade/src/api/services/FeatureFlagService.ts

```diff
@@ -407,6 +409,23 @@ export class FeatureFlagService {
     return this.segmentService.deleteSegment(segmentId, logger);
   }
 
+  public async validateForm(flagDTO: FeatureFlagValidation, logger: UpgradeLogger) {
```

Comment:
This validation should also happen during the `ImportFeatureFlag`

Reply:
```
if (keyExists) {
        compatibilityType = FF_COMPATIBILITY_TYPE.INCOMPATIBLE;
```

In import validation, we are sending as `Incompatible` if the key already exists